VoteNote/dbConnection.swift

```diff
@@ -266,13 +266,19 @@ func storePrevRoom(code: String){
             print("err")
         } else {
             
-            let rm = query?.documents[0].documentID
+            let rm = query?.documents[0]
+            
+            
             
             //if we didn't find the document
-            if rm == nil{
+            if rm?.documentID == nil{
                 print("err")
             } else {
-                db.collection("users").document(uid!).collection("prevRooms").addDocument(data: ["code": upperCode , "time": FieldValue.serverTimestamp()])
+                let isHost: Bool;
+                if rm!.data()["host"] as? String ?? "" == uid! {
+                    isHost = true
+                } else {isHost = false}
+                db.collection("users").document(uid!).collection("prevRooms").addDocument(data: ["code": upperCode , "time": FieldValue.serverTimestamp(), "isHost": isHost])
```

Comment:
does this update existing entries too (timestamp) or create a new entry each time?

Reply:
It does not, that implementation will be coming